.gitignore

```diff
@@ -103,3 +103,6 @@ venv.bak/
 # mypy
 .mypy_cache/
 /vulr
+
+# Secrets
+*config.yml
```

Comment:
I would probably suggest something like *secrets*.yml what do you think ?
Sometimes I call my yml here pipeline_config.yml or things like that

Reply:
Using the name `pipeline_config.yml` is fine and expected - here we're assuming that's the kind of name most people would use. gitignoring the assumedly most common suffix and instructing to use that suffix makes it safer in case anything in the config is considered a secret (e.g. `subscription_id`, for some people) or the config grows to include secrets.

I'm assuming we don't want to version control the actual config files in use, but could be wrong.